src/dataIntegration/transcriptomics/mapGeneToRxn.m

```diff
@@ -0,0 +1,64 @@
+function expressionRxns = mapGeneToRxn(model, gene_id, gene_expr, parsedGPR, corrRxn)

```

Comment:
This function can probably be combined with extractGPRs
And: I would open the discussion how gene expression should be mapped.
Maybe we can add options for how a gene expression is supposed to be mapped. I can imagine a couple of things:
E.g. if a gene is associated with multiple reactions, would its expression have to be split over the reactions? 
isn't an OR association an addition? I agree with AND being a minimum, but OR is difficult.